apps/backoffice-v2/src/pages/MerchantMonitoring/components/MerchantMonitoringTable/columns.tsx

```
@@ -28,6 +28,10 @@ import {
   WarningFilledSvg,
 } from '@ballerine/ui';
 import { Minus } from 'lucide-react';
+import {
+  MERCHANT_REPORT_STATUSES_MAP,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'MERCHANT_REPORT_STATUSES_MAP'. Consider to delete it or rename it.

'MERCHANT_REPORT_STATUSES_MAP' is defined here:


(lint/suspicious/noRedeclare)